tox.ini

```diff
@@ -88,6 +88,7 @@ whitelist_externals = bash
 # one day we'll use a fully pinned venv here...
 deps =
     urllib3<2.0
+    requests<2.32.0	# tmp fix for broken python docker: https://github.com/psf/requests/issues/6707
```

Comment:
i think we likely also want to pin requests in requirements-minimal.txt since i think there's other scripts that use dockerpy